delft/textClassification/models.py

```diff
@@ -285,7 +284,7 @@ def train_folds(X, y, model_config, training_config, embeddings, callbacks=None)
         val_x = X[fold_start:fold_end]
         val_y = y[fold_start:fold_end]
 
-        foldModel = getModel(model_config, training_config)
+        foldModel = getModel(model_config, training_config, print_summary=fold_id == 0)
```

Comment:
I find it hard to read... probably pythonesque and subjective, but I really don't like one-liner for maintenance and when taking over old code.